docs: update azure postgres mysql (#48726)
gravitational/teleport: docs/pages/enroll-resources/database-access/enroll-azure-databases/azure-postgres-mysql.mdx

You can create multiple database users identified by the same service principal.

## Step 4/5. Install the Teleport Database Service

(!docs/pages/includes/tctl-token.mdx serviceName="Database" tokenType="db" tokenFile="/tmp/token"!)

Install Teleport on the host where you will run the Teleport Database Service:

(!docs/pages/includes/install-linux.mdx!)

Create the Database Service configuration.

<Tabs>
<TabItem label="PostgreSQL">

- Specify the region for your database(s) in `--azure-postgres-discovery`.

> - Replace the `--proxy` value with your Teleport proxy address or Teleport cloud

Reviewer: ```suggestion
- Replace the `--proxy` value with your Teleport Proxy Service address or Teleport Enterprise (Cloud)
```
For consistent terms in the docs

>   URI (e.g. `mytenant.teleport.sh:443`):

Reviewer: ```suggestion
  URI (e.g. `example.teleport.sh:443`):
```

  ```code
  $ sudo teleport db configure create \
    -o file \
    --proxy=teleport.example.com:443 \
    --token=/tmp/token \
    --azure-postgres-discovery=eastus
  ```

</TabItem>
<TabItem label="MySQL">

- Specify the region for your database(s) in `--azure-mysql-discovery`.

> - Replace the `--proxy` value with your Teleport proxy address or Teleport cloud
>   URI (e.g. `mytenant.teleport.sh:443`):

Reviewer: ```suggestion
- Replace the `--proxy` value with your Teleport Proxy Service address or Teleport Enterprise (Cloud)
  URI (e.g. `example.teleport.sh:443`):
```

  ```code
  $ sudo teleport db configure create \
    -o file \
    --proxy=teleport.example.com:443 \
    --token=/tmp/token \
    --azure-mysql-discovery=eastus
  ```

</TabItem>

<TabItem label="PostgreSQL and MySQL">

Run the following command on your Database Service host:

  ```code
  $ sudo teleport db configure create \
    -o file \
    --proxy=teleport.example.com:443 \
    --token=/tmp/token \
    --azure-mysql-discovery=eastus \
    --azure-postgres-discovery=eastus
  ```

> <Admonition type="tip">

Reviewer: I think we can remove the `Admonition` tags, since the Admonition is the only text below the code snippet. Otherwise, I think this is too many boxes in one place.
